unlock-app/src/middlewares/currencyConversionMiddleware.js

```diff
@@ -2,19 +2,22 @@
 /* eslint promise/prefer-await-to-then: 0 */
 
 import axios from 'axios'
-
 import { setConversionRate } from '../actions/currencyConvert'
+import configure from '../config'
+
+const { services } = configure(global)
+const currencyPriceLookupURI = services.currencyPriceLookup
```

Comment:
It's up to you which syntax you prefer, but you can do this as a one liner with:
```
const { services: { currencyPriceLookupURI } } = configure(global)
```

LGTM!